main.py

```diff
@@ -18,29 +18,29 @@ def __init__(self, defaultlang):
     def _(self, text):
```

Comment:
optional extension: by re-declaring the method `_` into `__call__` one could instantiate the `Lang` class directly to a variable with the name `_` like so `_ = Lang("en")` and with that the `tr._("translate me")` could be replaced by `_("translate me"), and use it like in the original gettext C implementation.